### DISCUSSION TITLE
What order are items returned in a Catalog Items API search?

### DISCUSSION THREAD

**Discussion by NA-Dev** [amzn/selling-partner-api-models#3559] [General]
When I make a call to /catalog/2020-12-01/items/ with a search term, what logic is used to order the results returned? Do they roughly match organic results what would display on Amazon.com if I search for that term?

I am looking for a way to keep track of how far up my items are on the page when a term is searched on Amazon.com without having to web scrape or use 3rd party tools.

**Comment by chriskshipcor** [comment]
I'm trying to do the same thing. It looks like the searchCatalogItems will is mostly based on searching the text of the title. I've had examples of a product coming up number one on the searchCatalogItems response but not showing up at all when actually searching the same key terms on Amazon. Going back to MWS, the ListMatchingProducts seems to work better but it only shows the first 10 results which doesn't help too much either. If you've found something or if I'm missing something let me know, it would be great to be able to get the results of the actual search results on Amazon through the API.
